TaskSharper.Calender.WPF/ViewModels/MainWindowViewModel.cs

```diff
@@ -42,6 +44,14 @@ public MainWindowViewModel(IRegionManager regionManager, IEventAggregator eventA
             _logger = logger.ForContext<MainWindowViewModel>();
             _statusRestClient = statusRestClient;
 
+            _notificationSound = new MediaElement
+            {
+                LoadedBehavior = MediaState.Manual,
+                UnloadedBehavior = MediaState.Manual,
+                Source = new Uri("Media/WindowsNotifyCalendar.wav", UriKind.Relative),
+                Volume = 10
```

Comment:
Can we have this in a config file instead?

Reply:
The proposition of this controversy have been accomplished.